src/applications/static-pages/dependency-verification/components/dependencyVerificationModal.jsx

```diff
@@ -1,20 +1,30 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
+import { connect } from 'react-redux';
 import Modal from '@department-of-veterans-affairs/component-library/Modal';
+import { dependencyVerificationCall } from '../actions/index';
 import DependencyVerificationHeader from './dependencyVerificationHeader';
 import DependencyVerificationList from './dependencyVerificationList';
 import DependencyVerificationFooter from './dependencyVerificationFooter';
 
-const DependencyVerificationModal = () => {
+const DependencyVerificationModal = props => {
   const [isModalShowing, setIsModalShowing] = useState(false);
   const handleClick = e => {
```

Comment:
Do we actually need the event here? I think we could just go with something like 
`handleClick = () => setIsModalShowing(prevState => !prevState)` 
and then assign it to the listener like `onClose={handleClick}`

Reply:
yep, updated this